ModTekInjector/Injector.cs

```diff
@@ -293,6 +293,10 @@ private static int Main(string[] args)
             {
                 SayException(e);
             }
+            finally
+            {
+                ReadLine();
```

Comment:
There is nothing keeping the application open except for the console's default behavior. We can do something like:

```cs
catch (BackupFileNotFound e)
{
    SayException(e);
    SayHowToRecoverMissingBackup(e.BackupFileName);
    PromptForKey(true);
    return RC_MISSING_BACKUP_FILE;
}
catch (BackupFileInjected e)
{
    SayException(e);
    SayHowToRecoverInjectedBackup(e.BackupFileName);
    PromptForKey(true);
    return RC_BACKUP_FILE_INJECTED;
}
catch (Exception e)
{
    SayException(e);
    PromptForKey(true);
}
```

This would force the issue for exceptions, ignoring the option for requiring keypresses. Thoughts?

Reply:
Sounds good